docs/requirements.txt

```diff
@@ -3,5 +3,6 @@ sphinx-rtd-theme>=1.0.0
 numpydoc>=1.2
 sphinx-codeautolink>=0.10
 matplotlib>=3.4.3
+more_itertools>=8.13.0
```

Comment:
these are the (extra) requirements for producing the documentation. more_itertools is not required here (but please add in setup.py)